[INFO   ] Logger: Record log in /sdcard/others/codeProjects/ajedrezArtificial/.kivy/logs/kivy_18-08-31_22.txt
[INFO   ] Kivy: v1.10.1.dev0, git-ae3665c, 20180425
[INFO   ] Python: v3.6.2 (default, Apr 25 2018, 09:37:02) 
[GCC 7.2.0]
[INFO   ] Factory: 194 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/sdcard/others/codeProjects/ajedrezArtificial/tableroForm.py", line 30, in <module>
[WARNING] stderr:     TableroApp().run()
[WARNING] stderr:   File "/data/data/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.6/site-packages/kivy/app.py", line 800, in run
[WARNING] stderr:     root = self.build()
[WARNING] stderr:   File "/sdcard/others/codeProjects/ajedrezArtificial/tableroForm.py", line 27, in build
[WARNING] stderr:     return TableroScreen()
[WARNING] stderr:   File "/sdcard/others/codeProjects/ajedrezArtificial/tableroForm.py", line 15, in __init__
[WARNING] stderr:     super(LoginScreen, self).__init__(** kwargs)
[WARNING] stderr: NameError: name 'LoginScreen' is not defined
